pkg/kube/servicewatcher.go

```diff
@@ -101,15 +104,16 @@ func watchServices(ctx context.Context, client *kubernetes.Clientset) (<-chan ev
 	informerFactory.WaitForCacheSync(ctx.Done())
 	informerFactory.Start(ctx.Done())
 
-	services, err := serviceInformer.Lister().List(labels.Everything())
+	services, err := client.CoreV1().Services(corev1.NamespaceAll).List(ctx, v1.ListOptions{})
```

Comment:
Discussed with Nino: This manages to return an error (good!), but we don't handle it (bad!) and exit the whole process. At least `init` restarts us, but we should avoid needing to do that.